lib/Doctrine/Sniffs/Operators/OperatorSpacingSniff.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Sniffs\Operators;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\OperatorSpacingSniff as SquizOperatorSpacingSniff;
+use PHP_CodeSniffer\Util\Tokens;
+use function in_array;
+use const T_ARRAY_CAST;
+use const T_BOOL_CAST;
+use const T_DOUBLE_CAST;
+use const T_ECHO;
+use const T_INT_CAST;
+use const T_MINUS;
+use const T_OBJECT_CAST;
+use const T_PLUS;
+use const T_PRINT;
+use const T_STRING_CAST;
+use const T_UNSET_CAST;
+use const T_YIELD;
+
+/**
+ * We need this sniff until Squiz accepts fix for unary operands detection
+ * https://github.com/squizlabs/PHP_CodeSniffer/pull/2640
+ */
+final class OperatorSpacingSniff extends SquizOperatorSpacingSniff
+{
+    private const NON_OPERAND_TOKENS = [
+        T_ECHO,
+        T_PRINT,
+        T_YIELD,
+
```

Comment:
```suggestion
```

Reply:
Really? I think the grouping makes sense ... but if you insist, I'll remove it.